src/applications/vaos/referral-appointments/redux/actions.js

```diff
@@ -34,6 +42,90 @@ export function setFormCurrentPage(currentPage) {
   };
 }
 
+export function createReferralAppointment({
```

Comment:
This is a good way to do the check status we might have to think about the frequency a bit. I think we need to set up another function for the initial call as I remember we were talking about separate endpoints. We can do that in another ticket.